doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -38,11 +38,11 @@ While this Reference Architecture is being developed, Gaps will be identified th
 
 
 <a name="6.2.2"></a>
-### 6.2.2 Multi-tenancy within Kubernetes
+### 6.2.2 Multi-tenancy and workload isolation with Kubernetes
 
-> Today, Kubernetes lacks hard multi-tenancy capabilities<sup>citations</sup>
-
-> Ability to allow untrusted tenants to share infrastructure resources.
+**Related requirements:** `e.man.004` `sec.ci.008` `sec.wl.005``sec.wl.006` `req.inf.ntw.03` 
+**Baseline project:** none
+**Gap description:** Today, Kubernetes lacks hard multi-tenancy capabilities that give the ability to allow untrusted tenants to share infrastructure resources. This presents a security problem when operators seek to separate workloads by categorization or even just production vs non-production. Further, tenant networks need to be both segregated, but still centrally administered and maintained. Beyond just security, this also presents an operational problem. Trying to deploy too many CNFs into the same cluster could result in version conflicts, configuration conflicts, and problems with software life cycle management. Finally, without proper isolation there is an increased risk of cascading failures. 
```

Comment:
is "untrusted" needed?

What does "this" refer to in: "Beyond just security, this also presents an operational problem."

Since "workload" is used within the second sentence, can we replace "CNFs" by "workloads" in "Trying to deploy too many CNFs into the same cluster ..."